Konnect teams and roles (#3781)
Kong/docs.konghq.com: app/konnect/org-management/deactivation.md

* The control plane associated with the organization is decommissioned.

> * Any users that were part of the organization lose membership to any teams

Reviewer: I had to read this a couple of times.

```suggestion
* Any users that were part of the organization are removed from any teams
```
